docs/api/endpoints.md

```diff
@@ -186,21 +226,46 @@ __Response__
 PATCH /api/me (requires authentication)
 ```
 
-__Example__
+__NOTE__
+- Complete resource goes in request but a user itself is not allowed to update `read_only_fields` viz. `id`, `ticket_id`, `is_core_organizer`, `is_volunteer`, `date_joined`, `is_active`, `is_staff`, `is_superuser`
```

Comment:
The complete resource doesn't go into the request. Even if it goes, everything except read_only_fields can be updated.